packages/@react-stately/color/src/useColorSliderState.ts

```diff
@@ -51,19 +51,35 @@ export function useColorSliderState(props: ColorSliderStateOptions): ColorSlider
       setColor(color.withChannelValue(channel, v));
     },
     onChangeEnd([v]) {
-      // onChange will have already been called with the right value, this is just to trigger onChangEnd
+      // onChange will have already been called with the right value, this is just to trigger onChangeEnd
       if (props.onChangeEnd) {
         props.onChangeEnd(color.withChannelValue(channel, v));
       }
     }
   });
 
+  function incrementThumb(index: number, stepSize: number = 1) {
+    let {maxValue, minValue, step} = color.getChannelRange(channel);
+    let v = color.getChannelValue(channel);
+    let s = Math.max(stepSize, step);
+    sliderState.setThumbValue(index, v + s > maxValue ? maxValue : snapValueToStep(v + s, minValue, maxValue, s));
+  }
+
+  function decrementThumb(index: number, stepSize: number = 1) {
+    let {maxValue, minValue, step} = color.getChannelRange(channel);
+    let v = color.getChannelValue(channel);
+    let s = Math.max(stepSize, step);
+    sliderState.setThumbValue(index, v - s < minValue ? minValue : snapValueToStep(v - s, minValue, maxValue, s));
```

Comment:
I don't think page step should snap to page step size, instead, it should snap to the actual step size, what do you think?
```suggestion
    sliderState.setThumbValue(index, v - s < minValue ? minValue : snapValueToStep(v - s, minValue, maxValue, step));
```

Reply:
In that case, if you increment 3 by 16 it will snap to 19 rather than 16, is this how you think it should behave? Or is it better for color channels to snap to 8-bit sorts of values (16. 32. 48, 64, …)?

Reply:
My expectation for `snapValueToStep` is for values to snap to whichever step you are using, i.e. when incrementing by page size, snap to page size.

Reply:
That's not how steps work with a native slider. Page step will snap to step values, not to page step values.
Sorry for brevity, on my phone

Reply:
Since we are using separate parameters for `step` and `pageStep`, I think comparing how page step behaves in a ColorSlider versus a native slider is kind of a moot point. Of course snapping to `pageStep` is similarly arbitrary, I just think it results in more logical HEX values for colors, making it a bit easier to read and understand how far one is incrementing or decrementing. I page step in a native slider seems to always try to divide the range evenly by 10 while snapping to step.